src/site/content/en/blog/goibibo/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: How Goibibo's PWA improved conversions by 60%
+subhead: Bridging the gap between web and iOS/Android experiences to delight users.
+date: 2020-09-29
+# hero: TODO
+description: >
+  Learn how Goibibo, India’s top online travel company, achieved a 60% increase in
+  conversions by building reliable user experiences between their PWA and iOS/Android apps. 
+tags:
+  - blog
+  - case-study
+  - capabilities
+  - progressive-web-apps
+  - scale-on-web
+---
+
+[Goibibo](https://goibibo.com) is India's leading online travel booking portal. By building a full-featured
+and reliable [Progressive Web App](/pwa) that matched the capabilities of their iOS and Android
+apps, Goibibo achieved a 60% increase in conversions (compared to their previous web flow).
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled w-figure--inline-right"
+       src="mobile.png" 
+       alt="Goibibo's PWA.">
+</figure>
+
+<div class="w-stats">
+  <div class="w-stat">
+    <p class="w-stat__figure">60<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Increase in conversions</p>
+  </div>
+  <div class="w-stat">
+    <p class="w-stat__figure">20<sub class="w-stat__sub">%</sub></p>
+    <p class="w-stat__desc">Increase in logged-in users</p>
+  </div>
+</div>
+
+<br>
```

Comment:
This br seems to add a fair amount of white space which looks a bit odd on mobile. I removed it with devtools and thought it looked a bit better.

Reply:
SG; IIRC those `<br>` elements were leftover from my earlier tinkerings